public/docs/_examples/dependency-injection/dart/lib/car/car_no_di.dart

```diff
@@ -0,0 +1,20 @@
+// Car without DI
+import 'car.dart';
+
+//#docregion car
+class Car {
+  //#docregion car-ctor
+  Engine engine;
+  Tires tires;
+  var description = 'No DI';
+  Car() {
+    engine = new Engine();
+    tires = new Tires();
+  }
+  //#enddocregion car-ctor
+
+  // Method using the engine and tires
+  drive() => '${description} car with '
```

Comment:
Oh, this string is closer to what I recommended above! But `${description}` can be just `$description`, and we can delete the space before `tires.make`.